commands/qemu-guest-tools/actions_test.go

```diff
@@ -0,0 +1,261 @@
+package qemuguesttools
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"path/filepath"
+	goruntime "runtime"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
```

Comment:
I just let it do what my `go fmt` does automatically... maybe we should check that we're running the same formatting tools :)

And latest version of said formatting tools... I'll admit my editor runs a bunch, so I'm not even 100% sure which I'm using... But I would guess `go fmt`...